app/packages/app/src/recoil/aggregations.ts

```diff
@@ -132,6 +132,7 @@ export const aggregations = selectorFamily<
     }
 
     const dataset = get(selectors.datasetName);
+    get(aggregationsTick);
```

Comment:
I see this elsewhere in this file, but it's not clear why. Does this force an update or linking of some sort?

Reply:
Yes, it forces an update. It could/should be removed. Just need to check behavior doesn't change.